### PR TITLE
Merge shared changes

### PR DESCRIPTION
**Pull request by millerds**
Thank you for your pull request!  Please provide the following information.

---

**Change Description**:
Get changes from the base taskpane repo and merge them here

1. **Do these changes impact any *npm scripts* commands (in package.json)?** (e.g., running 'npm run start')
No.

2. **Do these changes impact *VS Code debugging* options (launch.json)?**
No.

3. **Do these changes impact *template output*?** (e.g., add/remove file, update file location, update file contents)
Some manifest updates

4. **Do these changes impact *documentation*?** (e.g., a tutorial on https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins)
I don't believe so

If you answered yes to any of these please do the following:
    > Include 'Rick-Kirkham' in the review
    > Make sure the README file is correct

**Validation/testing performed**:
Ran automated tests.

### CODE REVIEW

**Review by Adrian-MSFT — APPROVED** [review]

File: webpack.config.js [hunk 1]
```diff
@@ -52,7 +49,7 @@ module.exports = async (env, options) => {
           use: "html-loader",
         },
         {
-          test: /\.(png|jpg|jpeg|ttf|woff|woff2|gif|ico)$/,
+          test: /\.(png|jpg|jpeg|gif|ico)$/,
```
Comment (Rick-Kirkham) [inline webpack.config.js:52]: David found that in some scenarios ttf, woff, and woff2 need to be included.

```suggestion
          test: /\.(png|jpg|jpeg|ttf|woff|woff2|gif|ico)$/,
```

File: manifest.outlook.xml [hunk 2]
```diff
@@ -49,11 +49,11 @@
         <Host xsi:type="MailHost">
           <DesktopFormFactor>
             <FunctionFile resid="Commands.Url" />
-            <ExtensionPoint xsi:type="MessageComposeCommandSurface">
+            <ExtensionPoint xsi:type="MessageReadCommandSurface">
```
Comment (Rick-Kirkham) [inline manifest.outlook.xml:52]: Don't make this change. The function in the task pane writes to the email, so it has to be in compose mode.

```suggestion
            <ExtensionPoint xsi:type="MessageComposeCommandSurface">
```

File: manifest.outlook.xml [hunk 3]
```diff
@@ -49,11 +49,11 @@
...
               <OfficeTab id="TabDefault">
-                <Group id="msgComposeGroup">
+                <Group id="msgReadGroup">
```
Comment (Rick-Kirkham) [inline manifest.outlook.xml:54]:
```suggestion
                <Group id="msgComposeGroup">
```

File: manifest.outlook.xml [hunk 4]
```diff
@@ -49,11 +49,11 @@
...
                   <Label resid="GroupLabel" />
-                  <Control xsi:type="Button" id="msgComposeOpenPaneButton">
+                  <Control xsi:type="Button" id="msgReadOpenPaneButton">
```
Comment (Rick-Kirkham) [inline manifest.outlook.xml:56]:
```suggestion
                  <Control xsi:type="Button" id="msgComposeOpenPaneButton">
```

File: manifest.outlook.xml [hunk 5]
```diff
@@ -87,9 +102,10 @@
         <bt:ShortStrings>
           <bt:String id="GroupLabel" DefaultValue="Contoso Add-in"/>
           <bt:String id="TaskpaneButton.Label" DefaultValue="Show Task Pane"/>
+          <bt:String id="ActionButton.Label" DefaultValue="Perform an action"/>
         </bt:ShortStrings>
         <bt:LongStrings>
-          <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Opens a task pane."/>
+          <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Opens a pane displaying all available properties."/>
```
Comment (Rick-Kirkham) [inline manifest.outlook.xml:108]: The functionality in the React task pane is different from the standard. 
 
```suggestion
          <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Opens a pane that enables users to insert text."/>
```

File: manifest.word.xml [hunk 6]
```diff
@@ -72,13 +72,13 @@
...
-        <bt:String id="TaskpaneButton.Label" DefaultValue="Show Task Pane"/>
+        <bt:String id="GetStarted.Title" DefaultValue="Get started with your sample add-in!" />
+        <bt:String id="CommandsGroup.Label" DefaultValue="Commands Group" />
+        <bt:String id="TaskpaneButton.Label" DefaultValue="Show Task Pane" />
       </bt:ShortStrings>
       <bt:LongStrings>
-        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started."/>
-        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane"/>
+        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
+        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane" />
```
Comment (Rick-Kirkham) [inline manifest.word.xml:81]: This change is reintroducing a mistake. It always should have been two words. It was one word for years but apparently, it got fixed at some point.

```suggestion
        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane" />
```

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 1 above)
Comment (millerds) [reply webpack.config.js:52]: Is that specific to react?  They aren't in the other template repos.  The more we can maintain uniformity between them the easier it is to maintain.

Comment (Rick-Kirkham) [reply webpack.config.js:52]: > Is that specific to react? They aren't in the other template repos. The more we can maintain uniformity between them the easier it is to maintain.

Specific to React. It's a Fluent UI issue: [[node module error when creating outlook web add-in](https://stackoverflow.com/questions/78072626/node-module-error-when-creating-outlook-web-add-in)](https://stackoverflow.com/questions/78072626/node-module-error-when-creating-outlook-web-add-in)

Comment (millerds) [reply webpack.config.js:52]: I see . . . we'll keep it here, but I suggest making the same change to the taskpane template (and we'll merge it across) so we are consistent and that become one less problem developers hit if they move to use the newer fluent libraries.

**Review by Rick-Kirkham — APPROVED** [review]

**Pull request merged**